frontend/js/components/map.js

```diff
@@ -139,15 +138,11 @@ export class LeafletMap extends React.Component {
             return <IssueMarker {...props} />
         })
 
-        let extents = geojsonExtent(JSON.parse(JSON.stringify(geojson)));
```

Comment:
I can't see how my changes could affect the map locations since it's all about the calculation of the map bounds and do not touch the markers at all.

`fetchIssuesIfNeeded` looks like a weird redux action to me. It selects the issue, fetches it again and then fetches all issues again. There isn't even any `if` in there and I don't think a redux action should contain one. Shouldn't it be sufficient to only raise `selectIssue` when a users click a marker? If my understanding of redux is correct, there should be separate `commentDeleted` and `commentAdded` actions if there is a change in the comment list of the issue. As a quickfix, maybe it's possible to dispatch `loadComments` instead of `fetchIssues` in `fetchIssueIfNeeded` and to write a reducer for it (there currently is none for RECEIVE_COMMENTS) in order to just reload the comments?